Common/AR/source/ArcGISArViewInterface.cpp

```diff
@@ -378,7 +364,7 @@ void ArcGISArViewInterface::geometryChanged(const QRectF& newGeometry, const QRe
                                                  Qt::InvertedLandscapeOrientation | Qt::InvertedPortraitOrientation);
   }
 
-  QQuickItem::geometryChanged(newGeometry, oldGeometry);
+  QQuickFramebufferObject::geometryChanged(newGeometry, oldGeometry);
```

Comment:
Should this just be 
```suggestion
  emit geometryChanged(newGeometry, oldGeometry);
```

Reply:
This not a signal, this is an overridden function.